save-cosv/src/main/kotlin/com/saveourtool/save/backend/service/IBackendService.kt

```diff
@@ -30,4 +32,17 @@ interface IBackendService {
      * @return found [UserPermissions] by organizationName
      */
     fun getUserPermissionsByOrganizationName(authentication: Authentication, organizationName: String): UserPermissions
+
+    /**
+     * @param authentication
+     * @param organizationName name of organization
+     * @param permission
+     * @return true if [authentication] has [permission] in [organizationName], otherwise -- false
+     */
+    @Blocking
+    fun hasPermissionInOrganization(
```

Comment:
I thought that Vlad made some custom endpoint for each user and this user contains everything

Reply:
yes and I use it. this interface is a bridge to use backend methods from save-cosv module which is part of backend